<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rightWidget</class>
 <widget class="QWidget" name="rightWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="EDIT_ACTIVITY">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="actName">
         <property name="text">
          <string>Activity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="relativeDateCheckBox">
         <property name="text">
          <string>Relative Date</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QCheckBox" name="minStartCheckBox">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Min Start Date</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="minStart">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="dateTime">
                <datetime>
                 <hour>0</hour>
                 <minute>0</minute>
                 <second>0</second>
                 <year>2000</year>
                 <month>1</month>
                 <day>2</day>
                </datetime>
               </property>
               <property name="time">
                <time>
                 <hour>0</hour>
                 <minute>0</minute>
                 <second>0</second>
                </time>
               </property>
               <property name="displayFormat">
                <string>yyyy-M-d</string>
               </property>
               <property name="calendarPopup">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QCheckBox" name="maxFinishCheckBox">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Max Finish Date</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="maxFinish">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="dateTime">
                <datetime>
                 <hour>0</hour>
                 <minute>0</minute>
                 <second>0</second>
                 <year>2000</year>
                 <month>1</month>
                 <day>2</day>
                </datetime>
               </property>
               <property name="time">
                <time>
                 <hour>0</hour>
                 <minute>0</minute>
                 <second>0</second>
                </time>
               </property>
               <property name="displayFormat">
                <string>yyyy-M-d</string>
               </property>
               <property name="calendarPopup">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QSpinBox" name="relativeYearMinStart"/>
             </item>
             <item>
              <widget class="QDateEdit" name="relativeMonthDayMinStart">
               <property name="displayFormat">
                <string>M-d</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QSpinBox" name="relativeYearMaxFinish"/>
             </item>
             <item>
              <widget class="QDateEdit" name="relativeMonthDayMaxFinish">
               <property name="displayFormat">
                <string>M-d</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="IterationLabel">
           <property name="text">
            <string>Itertion</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="maxIter">
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timeLagLabel">
           <property name="text">
            <string>time Lag</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="timeLag">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Outputs</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tableOutputParams">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="toolTip">
              <string/>
             </property>
             <property name="toolTipDuration">
              <number>-1</number>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="verticalHeaderShowSortIndicator" stdset="0">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string notr="true" comment="The variable name" extracomment="The variable name">Name</string>
              </property>
              <property name="toolTip">
               <string notr="true">name of the variable</string>
              </property>
              <property name="whatsThis">
               <string notr="true"/>
              </property>
             </column>
             <column>
              <property name="text">
               <string notr="true">Value</string>
              </property>
              <property name="toolTip">
               <string notr="true">dimension of the variable</string>
              </property>
              <property name="whatsThis">
               <string notr="true"/>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget_1">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Rules</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listRules"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="EDIT_PRECEDENCE">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Precedence Constraint : from</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="First">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="arrow">
           <property name="text">
            <string>To</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Second">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Type :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxType">
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string>Finish-&gt;Start</string>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Finish-&gt;Start</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Start-&gt;Start</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxMinLag">
         <property name="title">
          <string>Min. Timelag</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QSpinBox" name="spinBoxMinLag">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>365</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxMaxLag">
         <property name="title">
          <string>Max. Timelag</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QSpinBox" name="spinBoxMaxLag">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>365</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="NO_EDIT"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
